src/dataDisplay/table.jsx

```diff
@@ -1,21 +1,346 @@
+            resizableColumnWidthPercentage,
+        } = this.props;
+        const {
+            adjacentStickyColumnsTotalWidth,
+            stickyTableContainerWidth,
+        } = this.state;
+        const shouldHandleStickyBehavior = !fixed && !singleLine;
+        const parsedChildren = React.Children.map(children, (child) => {
+            if (child.type.name === TABLE_HEADER_TYPE) {
+                const tableHeaderRow = child.props.children;
+                const headerCells = tableHeaderRow.props.children;
+                const parsedHeaderCells = React.Children.map(
+                    headerCells,
+                    (headerCell, index) => {
+                        // eslint-disable-next-line max-len
+                        const isResizable = shouldHandleStickyBehavior &&
+                            (index === stickyColumnCount - 1 && index !== headerCells.length - 1);
+                        return {
+                            ...headerCell,
+                            props: {
+                                ...headerCell.props,
+                                isResizable,
+                                ...(isResizable && {
+                                    adjacentStickyColumnsTotalWidth,
+                                    stickyColumnCount,
+                                    stickyTableContainerWidth,
+                                    shouldResetWhiteSpaceStyle: shouldHandleStickyBehavior,
+                                    forceTableUpdate: this.forceTableUpdate,
+                                    // eslint-disable-next-line max-len
+                                    resizableColumnWidthPercentage,
+                                    // eslint-disable-next-line max-len
+                                    setResizableCellsWordWrapping: this.setResizableCellsWordWrapping,
+                                }),
+                            },
+                        };
+                    },
+                );
+                const parsedTableHeaderRow = {
+                    ...tableHeaderRow,
+                    props: {
+                        ...tableHeaderRow.props,
+                        children: parsedHeaderCells,
+                    },
+                };
+                return {
+                    ...child,
+                    props: {
+                        ...child.props,
+                        children: {
+                            ...parsedTableHeaderRow,
+                        },
+                    },
+                };
+            }
+            return child;
+        });
+        return parsedChildren;
+    }
+
+    applyStickyClassesToCells() {
+        const {
+            stickyColumnCount,
+            definition,
+            fullWidth,
+        } = this.props;
+        const stickyCells = this.tableRef.querySelectorAll(`.table--cell:nth-child(-n+${stickyColumnCount})`);
+        for (let i = 0; i < stickyCells.length; i += 1) {
```

Comment:
Same ... I'd also prefer some blank lines here.
```JavaScript
const {
    stickyColumnCount,
    definition,
    fullWidth,
} = this.props;

const stickyCells = this.tableRef.querySelectorAll(`.table--cell:nth-child(-n+${stickyColumnCount})`);

// eslint-disable-next-line no-plusplus
for (let i = 0; i < stickyCells.length; i++) {
    // ... do the loop thing ...
}

// do the next thing, but leave a blank line following the closing curly brace of the `for` loop ...
```

I tend to prefer a bit of whitespace to "let it breathe" a little and help readability.  But as I said, that might be just me. :-)

Reply:
I added the break lines, I'm used to work with blank spaces only after imports, propTypes/defaultTypes/function/class block scopes but let me know how do you think we could place them and for sure I can follow that 👍

Reply:
Per my own opinions/preferences, I tend to like blank lines before/after:
* Multi-line assignments/destructring
* Statements that create a nested block/scope (e.g. `if`, `switch`, `for`, `while`, etc.)

I feel it helps readability of these things.  Otherwise things are a bit "crowded" and run together.  

I'll note that this is not something we've officially stacked hands on as a team, so right now there's no standard, and no right or wrong way, other than what ESLINT enforces.

Reply:
Lookin' good.  Resolving.